src/world/area_arn/arn_05/events.c

```diff
@@ -320,7 +320,7 @@ StaticNpc N(npcGroup_8024223C)[] = {
         .id = NPC_BOO0,
         .settings = &N(npcSettings_8024172C),
         .pos = { 55.0f, 195.0f, 160.0f },
-        .flags = NPC_FLAG_PASSIVE | NPC_FLAG_100 | NPC_FLAG_LOCK_ANIMS | NPC_FLAG_NO_Y_MOVEMENT,
+        .flags = NPC_FLAG_PASSIVE | NPC_FLAG_100 | NPC_FLAG_LOCK_ANIMS | NPC_FLAG_JUMPING,
```

Comment:
I'm not a fan of this flag name change, can you explain why you called this 'jumping'? It doesn't make sense to me that an NPC is given a flag with this name in the group list a priori - is the NPC perpetually jumping?

Reply:
This flag, and only this flag, is always set when an enemy jumps and is cleared when their jump is complete.

Reply:
As for the previous name, NPC_FLAG_NO_Y_MOVEMENT, while an enemy is jumping under the control of an AI state function, its vertical position is governed by the AI instead of the physics engine. This is why the physics engine is instructed not to change the Y position.

Reply:
That makes sense. But why would this flag be set in a StaticNpc then? Is it immediately cleared?

Reply:
Yes. Nearly every AI main function I've looked at so far clears it on first call for grounded enemies and sets it for flying enemies, who always have it on. The only time it would not be immediately altered is if no AI is assigned.